staging-service/src/main/java/com/github/cafdataprocessing/services/staging/utils/JsonMinifier.java

```diff
@@ -59,18 +69,30 @@ public static final void validateAndMinifyJson(final InputStream inputStream,
         factory.configure(Feature.FLUSH_PASSED_TO_STREAM, false);
         factory.configure(Feature.AUTO_CLOSE_TARGET, false);
         final JsonParser parser = DocumentValidator.getValidatingParser(inputStream);
+        
+        Set<String> localRefFiles;
+
         try(final JsonGenerator gen = factory.createGenerator(outstream))
         {
             try
             {
-                processJsonTokens(parser, gen, storageRefPath, inprogressContentFolderPath, fieldValueSizeThreshold);
+                final ObjectMapper objectMapper = new ObjectMapper();
+                final JsonNode jsonNodes = objectMapper.readTree(parser);
```

Comment:
This defeats the whole purpose of the streaming, as it will read the entire stream into memory.